include/pressio/pressio_macros.hpp

```diff
@@ -51,30 +51,14 @@
 
 #include "pressio/ops_macros.hpp"
 
-// ----------------------------------------
-// logging macros
-// ----------------------------------------
-#define PRESSIO_LOG_LEVEL_TRACE	    0
-#define PRESSIO_LOG_LEVEL_DEBUG	    1
-#define PRESSIO_LOG_LEVEL_INFO	    2
-#define PRESSIO_LOG_LEVEL_WARN	    3
-#define PRESSIO_LOG_LEVEL_ERROR	    4
-#define PRESSIO_LOG_LEVEL_CRITICAL  5
-#define PRESSIO_LOG_LEVEL_OFF	    6
-
-// if we are in debug mode, enable debug prints by default
-#if !defined NDEBUG && !defined PRESSIO_ENABLE_DEBUG_PRINT
-#define PRESSIO_ENABLE_DEBUG_PRINT
-#endif
-
-#if defined(PRESSIO_ENABLE_DEBUG_PRINT) && !defined(PRESSIO_LOG_ACTIVE_MIN_LEVEL)
-// if DEBUG_PRINT is on but MIN_LEVEL off, then set min level to trace
-#define PRESSIO_LOG_ACTIVE_MIN_LEVEL	PRESSIO_LOG_LEVEL_TRACE
-#elif !defined(PRESSIO_ENABLE_DEBUG_PRINT) && !defined(PRESSIO_LOG_ACTIVE_MIN_LEVEL)
-// if DEBUG_PRINT=off and MIN_LEVEL=off, then set logging off
-#define PRESSIO_LOG_ACTIVE_MIN_LEVEL	PRESSIO_LOG_LEVEL_OFF
+#if PRESSIO_ENABLE_LOGGING
+    #include "pressio-log/core.hpp"
 #else
-//DEBUG_PRINT is off and LOG_ACTIVE_MIN_LEVEL=on, nothing to do
+    #define PRESSIOLOG_BASIC(...)   do {} while (0)
```

Comment:
what is pressiolog_basic? what does it mean? i think we should change the qualification "basic"